### PR TITLE
doc: content/en/docs/hertz/faq/_index.md

### PR DESCRIPTION
**Pull request by daydaygo**

### CODE REVIEW

File: content/en/docs/hertz/faq/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+title: "FAQ"
+linkTitle: "FAQ"
+weight: 4
+description: >
+
+---
+
+## High memory usage
+
+### unclosed client connection
```
Comment (Duslia) [inline content/en/docs/hertz/faq/_index.md:11]: 大写

File: content/en/docs/hertz/faq/_index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+If the Client side initiates a large number of connections without closing them, in extreme cases, there will be a great waste of resources, and with the increase of time, the memory utilization rate may be high.
+
+**Solution**
+
+Configure `idleTimeout` reasonably, and Hertz Server will turn off the connection after timeout to ensure the stability of the server side. The default configuration is 3 minutes.
+
+### Oversized request/response
+
+1. If the request and response are very large, and some other sending modes such as stream and chunk are not used, all the data will enter the memory, causing great pressure on the memory.
```
Comment (Duslia) [inline content/en/docs/hertz/faq/_index.md:21]: 这就直接说没有用 stream 和 chunk 就行了？ 应该不用突出“其他发送模式”了

Comment (Duslia) [reply content/en/docs/hertz/faq/_index.md:21]: 这句话还有点长，可以断个句

File: content/en/docs/hertz/faq/_index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+2. The streaming under netpoll network library is false streaming. Because netpoll uses LT trigger mode, when the data arrives, it will trigger netpoll to read the data; In terms of interface design, the Reader' interface is not implemented. In order to realize the streaming ability, Hertz encapsulates netpoll as Reader, but its own data still enters the memory uncontrollably, so it may cause memory pressure in the case of extremely large streaming requests.
```
Comment (Duslia) [inline content/en/docs/hertz/faq/_index.md:22]: fake streaming ?

**Comment by Duslia** [comment]
Perhaps the translation needs to pay attention to the following two points:
1. Software Translator is a tool, not god, do not use the result as the final result.
2. Translation principle: faithfulness, expressiveness and elegance.

**Comment by GuangmingLuo** [comment]
@daydaygo hi, will you continue to improve this PR ?

**Comment by GuangmingLuo** [comment]
This PR will be closed and the corresponding issue will be re-assigned if there is no reply till the end of today.

**Pull request closed without merging**